app/queues/certificate/certificate-flow.server.ts

```diff
@@ -28,6 +28,11 @@ interface AddCertRequest {
   username: string;
 }
 
+const { JOB_REMOVAL_FREQUENCY_MS } = process.env;
+
+// constant  for removing job on completion/failure (in milliseconds)
+const JOB_REMOVAL_INTERVAL_MS = 7 * 24 * 60 * 60 * 1000; // 7 days
```

Comment:
7 days seem very long. Does it have to be this long?

Reply:
I am not sure. I followed the suggested period in the issue description.